### PR TITLE
[WIP] [INFINITY-3440] Initial cleanup of various markdown files and old scripts.

### PR DESCRIPTION
**Pull request by jeid64**
Removed dcos-docker scripts as well as new-standalone-service.sh and
__init__.py. Cleaned up README and removed "stateful" from FAQ.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,8 +8,13 @@ Contributors Guide
 
 Discuss contributions to DC/OS SDK using GitHub [issues](https://github.com/mesosphere/dcos-commons/issues) and make contributions with GitHub [pull requests](https://github.com/mesosphere/dcos-commons/pulls).
 
-### Proposing Features
-If you would like to propose a new feature or a significant change, first open an issue to discuss it.  Project shepherds will provide feedback on the proposed change in terms of feasibility, design, and implementation guidance.  While all proposals are welcome, those that conflict with our roadmap or design principles may not be accepted.
+### Bugs / Feature Requests
+Think you’ve found a bug? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+* Login to the [Mesosphere JIRA](jira.mesosphere.com).  You will need a github or google account to use this service.
```
Comment (jeid64) [inline CONTRIBUTING.md]: GitHub + Google capitalized?

Comment (triclambert) [reply CONTRIBUTING.md]: Please reinstate the sentences about "shepherds" and "roadmap".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,8 +8,13 @@ Contributors Guide
 
 Discuss contributions to DC/OS SDK using GitHub [issues](https://github.com/mesosphere/dcos-commons/issues) and make contributions with GitHub [pull requests](https://github.com/mesosphere/dcos-commons/pulls).
 
-### Proposing Features
-If you would like to propose a new feature or a significant change, first open an issue to discuss it.  Project shepherds will provide feedback on the proposed change in terms of feasibility, design, and implementation guidance.  While all proposals are welcome, those that conflict with our roadmap or design principles may not be accepted.
+### Bugs / Feature Requests
+Think you’ve found a bug? Want to see a new feature? Please open a case in our issue management tool, JIRA:
+* Login to the [Mesosphere JIRA](jira.mesosphere.com).  You will need a github or google account to use this service.  
+* Navigate to the **DC/OS SDK project**.
```
Comment (triclambert) [inline CONTRIBUTING.md]: I think it's sufficient to link directly to eliminate this list of instructions, and just link directly to the project in question:  https://jira.mesosphere.com/projects/DCOS_SDK/issues

Comment (ToddGreenstein) [reply CONTRIBUTING.md]: I'll change the link to your suggestion.  Can we leave the workflow?  it's intentional and specific.  It also indicates that we no longer use Github issues, and provides context for why the sdk jira public is so anemic.

Comment (triclambert) [reply CONTRIBUTING.md]: I think the H is supposed to be capital.  I.e. GitHub.

**Comment by benclarkwood** [comment]
@keithchambers @triclambert @ToddGreenstein We good?

**Comment by benclarkwood** [comment]
@crackcode123 is there anything left to do for this ticket?

**Pull request merged**